hexa/datasets/queue.py

```diff
@@ -52,6 +52,33 @@ def download_file_as_dataframe(
         return pd.read_parquet(download_url)
 
 
+def metadata_profiling(file_content: pd.DataFrame) -> list:
+    for col in file_content.select_dtypes(include=["object"]).columns:
+        file_content[col] = file_content[col].astype("string")
+
+    data_types = file_content.dtypes.apply(str).to_dict()
```

Comment:
In the case of a CSV, will it be able to return a float / int / bool type based on the values found in the column ?

Reply:
```suggestion
    data_type = file_content.dtypes.apply(str).to_dict()
```

Reply:
Yes it is detecting:  float, int, bool and string.